Browserify returns empty module

When running `npm run build`, Browserify returns an empty module, instead of the library.

```
(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.urbitKeygen = f()}})(function(){var define,module,exports;return (function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){

},{}]},{},[1])(1)
});


```

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "nyc mocha --reporter spec test/test.js",
     "lint": "eslint src/",
-    "build": "mkdir -p dist && browserify src/index.js -s urbit-keygen > dist/index.js --verbose",
+    "build": "mkdir -p dist && browserify src/index.js -s urbit-keygen > dist/index.js",
     "prepublishOnly": "npm run build",
     "generateTestWallets": "node ./test/generate_test_wallets"
   },

--- a/package.json
+++ b/package.json
@@ -3,11 +3,13 @@
   "version": "0.16.1",
   "description": "Key derivation and HD wallet generation functions for Urbit.",
   "main": "src/index.js",
-  "browser": "dist/index.js",
+"browser": {
+  "src/index.js": "dist/index.js"
+},
   "scripts": {
     "test": "nyc mocha --reporter spec test/test.js",
     "lint": "eslint src/",
-    "build": "mkdir -p dist && browserify src/index.js -s urbit-keygen > dist/index.js",
+    "build": "mkdir -p dist && browserify src/index.js -s urbit-keygen > dist/index.js --verbose",
     "prepublishOnly": "npm run build",
     "generateTestWallets": "node ./test/generate_test_wallets"
   },

--- a/package.json
+++ b/package.json
@@ -1,11 +1,11 @@
 {
   "name": "urbit-key-generation",
-  "version": "0.16.1",
+  "version": "0.16.2",
   "description": "Key derivation and HD wallet generation functions for Urbit.",
   "main": "src/index.js",
-"browser": {
-  "src/index.js": "dist/index.js"
-},
+  "browser": {
+    "src/index.js": "dist/index.js"
+  },
   "scripts": {
     "test": "nyc mocha --reporter spec test/test.js",
     "lint": "eslint src/",

--- a/src/index.js
+++ b/src/index.js
@@ -7,7 +7,7 @@ const nacl = require('tweetnacl')
 const ob = require('urbit-ob')
 const secp256k1 = require('secp256k1')
 
-const { version } = require('../package.json')
+const { version, name } = require('../package.json')
 
 const GALAXY_MIN = 0x00000000
 const GALAXY_MAX = 0x000000ff
@@ -420,9 +420,11 @@ const generateWallet = async config => {
   const buf = Buffer.from(ob.patq2hex(ticket), 'hex')
 
   const meta = {
-    generatorName: `urbit-key-generation`,
-    generatorVersion: version,
-    proposalNumber: `UP8`,
+    generator: {
+      name: name,
+      version: version,
+    },
+    spec: "UP8",
     ship: ship,
     patp: patp,
     tier: tier,

--- a/src/index.js
+++ b/src/index.js
@@ -420,7 +420,9 @@ const generateWallet = async config => {
   const buf = Buffer.from(ob.patq2hex(ticket), 'hex')
 
   const meta = {
-    generator: `urbit-key-generation-v${version}`,
+    generatorName: `urbit-key-generation`,
+    generatorVersion: version,
+    proposalNumber: `UP8`,
     ship: ship,
     patp: patp,
     tier: tier,

--- a/test/assets/wallet0.json
+++ b/test/assets/wallet0.json
@@ -1,7 +1,7 @@
 {
  "meta": {
   "generatorName": "urbit-key-generation",
-  "generatorVersion": "0.16.1",
+  "generatorVersion": "0.16.2",
   "proposalNumber": "UP8",
   "ship": 1,
   "patp": "~nec",

--- a/test/assets/wallet0.json
+++ b/test/assets/wallet0.json
@@ -1,8 +1,10 @@
 {
  "meta": {
-  "generatorName": "urbit-key-generation",
-  "generatorVersion": "0.16.2",
-  "proposalNumber": "UP8",
+  "generator": {
+   "name": "urbit-key-generation",
+   "version": "0.16.2"
+  },
+  "spec": "UP8",
   "ship": 1,
   "patp": "~nec",
   "tier": "galaxy",

--- a/test/assets/wallet0.json
+++ b/test/assets/wallet0.json
@@ -1,6 +1,8 @@
 {
  "meta": {
-  "generator": "urbit-key-generation-v0.16.1",
+  "generatorName": "urbit-key-generation",
+  "generatorVersion": "0.16.1",
+  "proposalNumber": "UP8",
   "ship": 1,
   "patp": "~nec",
   "tier": "galaxy",

--- a/test/assets/wallet1.json
+++ b/test/assets/wallet1.json
@@ -1,6 +1,8 @@
 {
  "meta": {
-  "generator": "urbit-key-generation-v0.16.1",
+  "generatorName": "urbit-key-generation",
+  "generatorVersion": "0.16.1",
+  "proposalNumber": "UP8",
   "ship": 65012,
   "patp": "~matbyr",
   "tier": "star",

--- a/test/assets/wallet1.json
+++ b/test/assets/wallet1.json
@@ -1,8 +1,10 @@
 {
  "meta": {
-  "generatorName": "urbit-key-generation",
-  "generatorVersion": "0.16.2",
-  "proposalNumber": "UP8",
+  "generator": {
+   "name": "urbit-key-generation",
+   "version": "0.16.2"
+  },
+  "spec": "UP8",
   "ship": 65012,
   "patp": "~matbyr",
   "tier": "star",

--- a/test/assets/wallet1.json
+++ b/test/assets/wallet1.json
@@ -1,7 +1,7 @@
 {
  "meta": {
   "generatorName": "urbit-key-generation",
-  "generatorVersion": "0.16.1",
+  "generatorVersion": "0.16.2",
   "proposalNumber": "UP8",
   "ship": 65012,
   "patp": "~matbyr",

--- a/test/assets/wallet2.json
+++ b/test/assets/wallet2.json
@@ -1,8 +1,10 @@
 {
  "meta": {
-  "generatorName": "urbit-key-generation",
-  "generatorVersion": "0.16.2",
-  "proposalNumber": "UP8",
+  "generator": {
+   "name": "urbit-key-generation",
+   "version": "0.16.2"
+  },
+  "spec": "UP8",
   "ship": 222,
   "patp": "~ryl",
   "tier": "galaxy",

--- a/test/assets/wallet2.json
+++ b/test/assets/wallet2.json
@@ -1,6 +1,8 @@
 {
  "meta": {
-  "generator": "urbit-key-generation-v0.16.1",
+  "generatorName": "urbit-key-generation",
+  "generatorVersion": "0.16.1",
+  "proposalNumber": "UP8",
   "ship": 222,
   "patp": "~ryl",
   "tier": "galaxy",

--- a/test/assets/wallet2.json
+++ b/test/assets/wallet2.json
@@ -1,7 +1,7 @@
 {
  "meta": {
   "generatorName": "urbit-key-generation",
-  "generatorVersion": "0.16.1",
+  "generatorVersion": "0.16.2",
   "proposalNumber": "UP8",
   "ship": 222,
   "patp": "~ryl",

--- a/test/assets/wallet3.json
+++ b/test/assets/wallet3.json
@@ -1,6 +1,8 @@
 {
  "meta": {
-  "generator": "urbit-key-generation-v0.16.1",
+  "generatorName": "urbit-key-generation",
+  "generatorVersion": "0.16.1",
+  "proposalNumber": "UP8",
   "ship": 0,
   "patp": "~zod",
   "tier": "galaxy",

--- a/test/assets/wallet3.json
+++ b/test/assets/wallet3.json
@@ -1,7 +1,7 @@
 {
  "meta": {
   "generatorName": "urbit-key-generation",
-  "generatorVersion": "0.16.1",
+  "generatorVersion": "0.16.2",
   "proposalNumber": "UP8",
   "ship": 0,
   "patp": "~zod",

--- a/test/assets/wallet3.json
+++ b/test/assets/wallet3.json
@@ -1,8 +1,10 @@
 {
  "meta": {
-  "generatorName": "urbit-key-generation",
-  "generatorVersion": "0.16.2",
-  "proposalNumber": "UP8",
+  "generator": {
+   "name": "urbit-key-generation",
+   "version": "0.16.2"
+  },
+  "spec": "UP8",
   "ship": 0,
   "patp": "~zod",
   "tier": "galaxy",

--- a/test/assets/wallet4.json
+++ b/test/assets/wallet4.json
@@ -1,8 +1,10 @@
 {
  "meta": {
-  "generatorName": "urbit-key-generation",
-  "generatorVersion": "0.16.2",
-  "proposalNumber": "UP8",
+  "generator": {
+   "name": "urbit-key-generation",
+   "version": "0.16.2"
+  },
+  "spec": "UP8",
   "ship": 16777215,
   "patp": "~taswen-dabmyr",
   "tier": "planet",

--- a/test/assets/wallet4.json
+++ b/test/assets/wallet4.json
@@ -1,7 +1,7 @@
 {
  "meta": {
   "generatorName": "urbit-key-generation",
-  "generatorVersion": "0.16.1",
+  "generatorVersion": "0.16.2",
   "proposalNumber": "UP8",
   "ship": 16777215,
   "patp": "~taswen-dabmyr",

--- a/test/assets/wallet4.json
+++ b/test/assets/wallet4.json
@@ -1,6 +1,8 @@
 {
  "meta": {
-  "generator": "urbit-key-generation-v0.16.1",
+  "generatorName": "urbit-key-generation",
+  "generatorVersion": "0.16.1",
+  "proposalNumber": "UP8",
   "ship": 16777215,
   "patp": "~taswen-dabmyr",
   "tier": "planet",

--- a/test/generate_test_wallets.js
+++ b/test/generate_test_wallets.js
@@ -1,5 +1,5 @@
 const fs = require('fs')
-const kg = require('../dist/index')
+const kg = require('../src/index')
 
 const write = (wal, num) => {
   fs.writeFile(

